content/frontend/react-relay/5-authentication.md

```diff
@@ -120,7 +120,7 @@ With that component in place, you can go and add a new route to your `react-rout
 
 Open `App.js` and update `render` to include the new route:
 
-```js{7}(path=".../hackernews-react-relay/src/components/App.js")
+```js{7}(path=".../hackernews-react-apollo/src/components/App.js")
```

Comment:
yes absolutely, somehow messed that up with an old commit.
should be fixed in the new commit now